README.md

```diff
@@ -154,7 +154,7 @@ kind: Subscription
 metadata:
   name: knative-kafka-operator-sub
   generateName: knative-kafka-operator-
-  namespace: knative-eventing
+  namespace: $OPERATOR_NS
```

Comment:
I think we can delete the `Namespace` and `OperatorGroup` resources above this, too, no?